networkx/algorithms/tests/test_efficiency.py

```diff
@@ -53,15 +53,15 @@ def test_global_efficiency_complete_graph(self):
 
     def test_local_efficiency_complete_graph(self):
         """
-        Test that the local efficiency for a complete graph should be one.
+        Test that the local efficiency for a complete graph with at least 3 nodes should be one. For a graph with only 2 nodes, the induced subgraph has no edges.
```

Comment:
This line should be wrapped so that [the PEP8 style guide](https://www.python.org/dev/peps/pep-0008/) of <80 characters/line  holds. There may be other pep8 considerations. 

You can read the style guide and/or install the pep8 style checker using ```pip install pep8```

Reply:
Changed.